docs/src/research/run_win_vm.md

```diff
@@ -32,14 +30,55 @@ You can run Windows 11 in a VM on Ghaf with NVIDIA Jetson Orin AGX. This method
+2. In Weston terminal, go to the directory with the Windows 11 image and run the VM using the following Ghaf script:
+
+    ```
+    cd /mnt
+    windows-launcher ./Win11_22H2_English_x64v2.iso
+    ```
+
+    > **WARNING:** Do not use **sudo** or the root user to run windows-launcher.
+
+3. When the VM starts booting press any key to boot from CD.
+4. In order to bypass Windows 11 system requirement hit Shift+F10 to open a command prompt and type regedit. In `HKEY_LOCAL_MACHINE\SYSTEM\Setup` create a new `LabConfig` key (with right click select New, Key, type LabConfig). In this key create a DWORD (32-bit) parameter with the name `BypassTPMCheck` and value `1`. Also create a similar value with the `BypassSecureBootCheck` name.
 
-    When you click it for the first time, you will see a file selection dialog. Once Windows 11 image has been selected, it saves the path to the `~/.config/windows-launcher-ui.conf` configuration file and launches the VM. Next time, the VM will be immediately launched with one click.
+    > **Warning:** when running Ghaf on a notebook it might be needed to press the Fn key with Shift+F10. Also sometimes when Shift+F10 is pressed the command window opens in the background and not visible. In order to bring it to foreground hit Alt+Tab one or more times.
 
-2. You can pass additional parameter to QEMU after the image name. For example:
+5. Install Windows 11 in the virtual machine.
+6. Windows 11 requires Internet access to finish the setup. To boot the VM without an Internet connection, open cmd with Shift+F10 and type `OOBE\BYPASSNRO`. It will restart configuration and next time there will be a new button “I don’t have internet“. Hit that button to skip connecting to network and continue the installation.
+7. After the installation is completed the script should be launched with the QCOW2 image:
+
+    ```
+    windows-launcher ./win11.qcow2
+    ```
+
+## Using UI to Launch Windows 11 VM
+
+1. Instead of running windows-launcher from the command line it is possible to launch the Windows 11 VM by clicking the corresponding icon in the Weston taskbar.
+2. When you click it for the first time, you will see a file selection dialog. Once Windows 11 image has been selected, it saves the path to the `~/.config/windows-launcher-ui.conf` configuration file and launches the VM. Next time, the VM will be immediately launched with one click.
+3. In order to use a different image instead of the saved one, delete the configuration file:
+
+   ```
+   rm ~/.config/windows-launcher-ui.conf
+   ```
+
+## Passing Addition Parameters to QEMU
+
+1. It is possible to pass additional parameters to QEMU when running windows-launcher from the command line.
```

Comment:
Let's make this not a numbered list, but regular sentences: each one as a new line with a blank line between them.